tests/theia_automation_lsp/drivers/drv_config.py

```diff
@@ -25,15 +26,15 @@
 DRIVERS = {
     "darwin": {
         "firefox": "https://github.com/mozilla/geckodriver/releases/download/v0.24.0/geckodriver-v0.24.0-macos.tar.gz",
-        "chrome": "https://chromedriver.storage.googleapis.com/77.0.3865.40/chromedriver_mac64.zip"
+        "chrome": "https://chromedriver.storage.googleapis.com/80.0.3987.106/chromedriver_mac64.zip"
```

Comment:
This can break the local run. Can you please verify it? You can ask @AgisilaosVasileiadis for help.

Reply:
You are right. I didn't test it with chrome.

Reply:
@grianbrcom Any news about that?

Reply:
Finally, I was able to run it. The previous driver could work only with chrome 77. I've got version 81 installed locally. This change is required.